locksmith/src/operations/stripeOperations.ts

```diff
@@ -0,0 +1,60 @@
+import { ethereumAddress } from '../types' // eslint-disable-line import/named, no-unused-vars
+import * as Normalizer from '../utils/normalizer'
+import { UserReference } from '../models/userReference'
+
+import { User } from '../models/user'
+import { StripeCustomer } from '../models/stripeCustomer'
+
+const Sequelize = require('sequelize')
+
+const { Op } = Sequelize
+
+/**
+ * Method, which, given a publicKey, returns the stripe token id
+ * This does a double look up as we changed how stripe token ids are stored (used to be in UserReferences and are now in their own table)
+ * @param publicKey
+ */
+export const getStripeCustomerIdForAddress = async (
+  publicKey: ethereumAddress
+) => {
+  const normalizedEthereumAddress = Normalizer.ethereumAddress(publicKey)
```

Comment:
There are a few things occurring in this method, it would be good to add some coverage around the branching to ensure that the customer id is received when it should be

Reply:
Coverage of this file is now 100% ;) Thanks!